tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -353,7 +354,7 @@ Verify Installed Library Version
     [Arguments]  ${lib}  ${ver}
     ${status}  ${value} =  Run Keyword And Warn On Failure  Run Cell And Check Output  !pip show ${lib} | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s", b[1], b[2]}'  ${ver}
     Run Keyword If  '${status}' == 'FAIL'  Log  "Expected ${lib} at version ${ver}, but ${value}"
-    [Return]    ${status}
+    [Return]    ${status}    ${value}
```

Comment:
As you are modifying `Run Cell And Get Output` you should run all tests that use this keyword, in case the change is breaking them

Reply:
I have modified `Run Cell And Check Output` and  it is not affecting other tcs as they are using it to just run cell

Reply:
If you do "Find in files..." `Run Cell And Get Output` you'll find 2 or 3 places were they are getting the returned param status. They could be affected now that we have 2 return params

Reply:
I have modified `Run Cell And Check Output` not `Run Cell And Get Output`